lib/url.h

```diff
@@ -34,7 +34,7 @@
 CURLcode Curl_init_do(struct Curl_easy *data, struct connectdata *conn);
 CURLcode Curl_open(struct Curl_easy **curl);
 CURLcode Curl_init_userdefined(struct Curl_easy *data);
-CURLcode Curl_dupset(struct Curl_easy * dst, struct Curl_easy * src);
```

Comment:
`Curl_dupset()` is referred to in a comment in `lib/urldata.h` as well which should be cleanup along with this removal.

Reply:
nice catch, will remove